### ISSUE TITLE
Low visibility/contrast on LLM submenu active items

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#7608]
## Description 

In the main LLM menu contrast on submenu items that are active is low and it can be difficult to distinguish which item is currently focused. The light mode contrast of the background and active item's hover color in light mode is 1.2:1 and in dark mode is 1.4:1.

We also have some user feedback we received around the new menu from user testing to add a little more context to this issue. 

[ZoomText](https://www.freedomscientific.com/products/software/zoomtext/) user: "The outline of the selection field, where I could choose and change the default model, was light grey on a white background, making it hard to distinguish. I suggest making the outline darker to improve visibility and ensure it is easy to see."

<img width="624" alt="Screenshot of the LLM submenu in light mode showing the light grey on white of the active item." src="https://github.com/user-attachments/assets/45f0075c-ae43-417d-861c-13c88531a888" />

<img width="628" alt="Screenshot of the LLM submenu in dark mode showing the dark grey on black of the active item." src="https://github.com/user-attachments/assets/c9b07a74-0cda-4f79-bffc-84f9a06e7235" />

## WCAG Criteria
Supporting SSC are [2.4.7: Focus Visible (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html) and  [1.4.11: Non-text Contrast (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast)

## Notes
Based on feedback in #6645 adding an horizontal indicator to the left margin is a good solution. 

![Image](https://github.com/user-attachments/assets/7c6db71d-0b2a-4f1b-be8e-b43464d9029d)

**Comment by danny-avila** [comment]
closed by #7714
